synapse/lib/certdir.py

```diff
@@ -24,6 +24,37 @@ def iterFqdnUp(fqdn):
 
 logger = logging.getLogger(__name__)
 
+# Create a cipher string that supports TLS 1.2 and TLS 1.3 ciphers which do
+# not use RSA. This has the effect of modifying the cipher list for a SSL
+# context in Python 3.8 and below. Python 3.10+ has a default cipher list
+# which drops several additional ciphers which are removed below.
+_ctx = ssl.create_default_context(purpose=ssl.Purpose.CLIENT_AUTH)  # type: ssl.SSLContext
+_ciphers = [c for c in _ctx.get_ciphers()
```

Comment:
This feels like it should be unrolled into a function like TLS_SERVER_CIPHERS = initTLSServerCiphers() rather than a 3 line list comprehension and using `del` in the module namespace.